heroku/cli
# refactor: migrate members/index to oclif/core

Migrates the `members` command to oclif.

## Testing
- Check out branch and run `yarn` and `yarn build`
- Run `./bin/run members -t hk-front-end-online-test-team`
    - If you do not have access to this app, let me know and I will add you
    - You should se Santi as a pending member
- Run `./bin/run members -t heroku-front-end`
    - There are no pending members on this team, so you should not see a "Status" column
-  Run `./bin/run members -t hk-front-end-online-test-team --pending`
   - You should only see Santi's entry in the table
- Run  `./bin/run members -t hk-front-end-online-test-team --json`
    - You should get json output
